src/dmd/traits.d

```diff
@@ -1488,6 +1488,11 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
                 }
             }
 
+            // https://issues.dlang.org/show_bug.cgi?id=20915
+            // skip version and debug identifiers
+            if (sm.isVersionSymbol() || sm.isDebugSymbol())
+                return 0;
```

Comment:
```suggestion
            if (sm.isDVCondition())
                return 0;
```

This is what I meant @NilsLankila . This way you go from 2 virtual calls (and 2 methods) to one.
Also you should made the `override` `final`.

Reply:
src/dmd/traits.d(1495): Error: no property `isDVCondition` for type `src/dmd.dsymbol.Dsymbol`

Reply:
And the two classes (`DebugSymbol` and `VersionSymbol`) are `final` so the override are.

Reply:
The symbols are not Condition... that cant work.

Reply:
Indeed, they are symbol equivalent, not the `Condition`, sorry for the confusion.